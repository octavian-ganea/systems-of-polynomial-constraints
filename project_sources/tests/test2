x^2 + y^2 - x - y + 0.5 = 0