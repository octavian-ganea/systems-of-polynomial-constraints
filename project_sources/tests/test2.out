Parsing file test2
Parsing succeded. Results:
List of all variables:
Vector(x, y)
List of all polynomials:
Vector((1.0 y^2 + -1.0 y + 0.5 ) + -1.0 x + 1.0 x^2)

List of desired signs : Vector(=)
Analysed polynomials: 
(1.0 y^2 + -1.0 y + 0.5 ) + -1.0 x + 1.0 x^2
vars: Vector(x, y)

Univariate projection (on real axis) contains these polynomials: 
1.0 y^2 + -1.0 y + 0.5 
4.0 y^2 + -4.0 y + 1.0 

There are 5 cells :
Map(x -> 0.0, y -> -0.500000001287698)
Map(x -> -0.5000000013150449, y -> 0.499999998712302)
Map(x -> 0.49999999868495515, y -> 0.499999998712302)
Map(x -> 1.499999998684955, y -> 0.499999998712302)
Map(x -> 0.0, y -> 1.499999998712302)
====== FINAL RESULT FOR test2 ================================
Map(x -> 0.49999999868495515, y -> 0.499999998712302)
====================================================

